clap_blocks/src/compactor.rs

```diff
@@ -27,15 +27,52 @@ pub struct CompactorConfig {
     )]
     pub write_buffer_partition_range_end: i32,
 
-    /// Percentage of least recent data we want to split to reduce compacting non-overlapped data
-    /// Must be between 0 and 100. Default is 100, which won't split the resulting file.
+    /// Max number of level-0 files we want to compact with level-1 each time
+    /// Default: 3
+    /// TODO: verify the theory that if the streaming works  as expected,
+    /// we do not need to limit this number
+    #[clap(
```

Comment:
I would recommend putting this TODO note somewhere other than the docstring - when placed here it will appear in the actual message shown to users when they run `influxdb_iox run compactor --help`

Perhaps something like
```suggestion
    // TODO: verify the theory that if the streaming works  as expected,
    // we do not need to limit this number
    #[clap(
```